### ISSUE TITLE
Which time zone to use for date boundaries for the vendor retail analytics reports?

### ISSUE THREAD

**Issue by moltar** [amzn/selling-partner-api-models#2573]
Does it matter which time zone is used for the `dataStartTime` and `dataEndTime` values?

Source: https://developer-docs.amazon.com/sp-api/docs/report-type-values#vendor-retail-analytics-reports

> Requests must include the reportPeriod, distributorView, and sellingProgram properties. Use the dataStartTime and dataEndTime parameters to specify the date boundaries for the report. The dataStartTime and dataEndTime values must correspond to valid first and last days in the specified reportPeriod. For example, dataStartTime must be a Sunday and dataEndTime must be a Saturday when reportPeriod=WEEK .

Thanks!

**Comment by mystery4f** [comment]
Hi,

  We're using the Analytics Reports API (createReport, https://developer-docs.amazon.com/sp-api/docs/report-type-values-analytics) with dataStartTime and dataEndTime like this:

  {
    "dataStartTime": "2026-04-29T00:00:00Z",
    "dataEndTime": "2026-04-29T00:00:00Z"
  }

  The documentation states that the time component is ignored and only the date portion is used for report period alignment. So in this case, the API returns the report for the date 2026-04-29.

  However, it's unclear whether this date 2026-04-29 is interpreted as:

  1. UTC date — the report covers 2026-04-29 in UTC
  2. Marketplace local date — the report covers 2026-04-29 in the marketplace's local timezone (e.g., America/Los_Angeles for US, Europe/London for UK, Asia/Tokyo for JP)

  This matters significantly. For example, for a UK marketplace (BST, UTC+1 in summer):

  - If we pass "2026-04-28T00:00:00Z" (which is 2026-04-27 23:00:00 BST), should we expect the report for 2026-04-28 or 2026-04-27?
  - In other words, should we convert our local date to UTC before sending, or send the local date directly?

  Could you clarify how the API resolves the timezone for the date portion? This would help us ensure we're requesting the correct report period.

  Thank you!
